driver/csiplugin/controllerserver.go

```diff
@@ -136,12 +136,23 @@ func (cs *ScaleControllerServer) createDirectory(scVol *scaleVolume, targetPath
 		return fmt.Errorf("unable to check if directory path [%v] exists in filesystem [%v]. Error : %v", targetPath, scVol.VolBackendFs, err)
 	}
 
+	glog.V(5).Infof("DEEBUG permissions %s", scVol.VolPermissions)
```

Comment:
Minor comment: Change DEEBUG to DEBUG. Rest looks good.

Reply:
Done